### PR TITLE
Update how-to-create-release-announcements.md

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

**Comment by amyblais** [comment]
- ~~Screenshot quality - resolution. Recommendations: Full size screenshots (no zoom-in, no resizing within GDoc), potentially storing in a TIF file.~~
 - Lock title even earlier in the process.
 - ~~Make sure we give letter grades on title and feature summary (already in the guidelines, will focus on this even more) - add to post where I give blog to Ian~~
 - Improve description of tick-tock releases - ask non-admin to review
 - ~~Goal of the title is to get admins to upgrade (doesn't always need to follow the guidelines) - will add to our guidelines~~
 - ~~Screenshot guideline related to showing themes and theme icons~~

File: operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-create-screenshots-and-gifs.md [hunk 1]
```diff
@@ -69,7 +72,7 @@ Use [GIF Brewery](https://gfycat.com/gifbrewery) to create GIFs. Ensure there ar
 
 #### Don't zoom past 100% and pixelated images
 
-When you need to show a particular part of the product, don't crop the image and zoom past 100% to show the functionality. Example of what not to do: 
+When you need to show a particular part of the product, don't crop the image and zoom past 100% to show the functionality.   Screenshots should be full-sized (no zoom-in, no resizing within GDoc), potentially stored in a TIF file. Example of what not to do:
```
Comment (jasonblais) [inline operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-create-screenshots-and-gifs.md]:
```suggestion
When you need to show a particular part of the product, don't crop the image and zoom past 100% to show the functionality. Screenshots should be full-sized (no zoom-in, no resizing within Google Doc), potentially stored in a TIF file. For optimal quality on Mac screenshots, [see this help article](https://graphicdesign.stackexchange.com/questions/30081/highest-quality-website-screenshot/30142#30142).

Example of what not to do: 
```

File: operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-create-release-announcements.md [hunk 2]
```diff
@@ -79,6 +80,7 @@ The Release Manager \(Amy Blais\) owns the release announcement, including the f
       1. Names should not be red-underlined \(if they are, add them to the dictionary prior to taking a screenshot\)
       2. Screenshot not surrounded by a border
       3. Below the screenshot, include a text version of their names in small font with a link to their GitHub handle
+   8. One or more people on the team put themselves in the shoes of a system admin responsible for upgrading Mattermost, read through the announcement out loud, clicked on the links, and feel we've met our High Standards LP
```
Comment (jasonblais) [inline operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-create-release-announcements.md]:
```suggestion
   8. One or more people on the team put themselves in the shoes of a System Admin responsible for upgrading Mattermost and have read through the announcement out loud, clicked on the links, and feel we've met our [High Standards leadership principle](https://handbook.mattermost.com/company/about-mattermost#leadership-principles)
```

**Review by jasonblais — APPROVED** [review]
For the two outstanding items, should we create a handbook issue to keep track of it? (I know you'll have it in your task list, but handbook issue gives more visibility)

**Review by jasonblais — CHANGES_REQUESTED** [review]
Meant to request changes

**Comment by amyblais** [comment]
https://github.com/mattermost/mattermost-handbook/issues/147 for other updates.

**Review by jasonblais — APPROVED** [review]

**Pull request merged**